packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -34,11 +35,22 @@ export class ApexTestRunner {
   }
 
   public async showErrorMessage(test: TestNode) {
```

Comment:
When a test class is marked as failed, the "Display Error" option will still show up upon right clicking it. It seemed confusing to have Display Error bring you to the top of the class file, so I changed it to go to the first failed test case.